pkg/quota/v1/evaluator/core/services.go

```diff
@@ -120,22 +122,39 @@ func (p *serviceEvaluator) Usage(item runtime.Object) (corev1.ResourceList, erro
 	// default service usage
 	result[serviceObjectCountName] = *(resource.NewQuantity(1, resource.DecimalSI))
 	result[corev1.ResourceServices] = *(resource.NewQuantity(1, resource.DecimalSI))
-	result[corev1.ResourceServicesLoadBalancers] = resource.Quantity{Format: resource.DecimalSI}
-	result[corev1.ResourceServicesNodePorts] = resource.Quantity{Format: resource.DecimalSI}
```

Comment:
with the zero-amount fix in the quota component, I think these lines should be restored